Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -398,10 +420,26 @@ theorem comap {S : Type*} [Ring S] (f : S →+* R) (h : v₁.IsEquiv v₂) :
     (v₁.comap f).IsEquiv (v₂.comap f) := fun r s => h (f r) (f s)
 #align valuation.is_equiv.comap Valuation.IsEquiv.comap
 
+/-- Restate the definition of `Valuation.IsEquiv` into a theorem. -/
```

Comment:
Why is this useful?

Reply:
Originally, these are lemmas for the typeclass `IsValExtension`, stating that if we have `IsValExtension R A`, we have `v (algebraMap R A r)  < v (algebraMap R A r') ↔ v r < v r'`, with a `@[simp]` label. (Same for `val_le_one`, `val_lt_one`,  `val_eq_one`) Then I was suggested that these lemmas should be stated in a more general form like this. What is your suggestion for these lemmas?

Reply:
This lemma and the following lemmas are used in the proof of 
```
theorem IsValExtension.val_map_le_iff (x y : R) : vA (algebraMap R A x) ≤ vA (algebraMap R A y) ↔ vR x ≤ vR y :=
  (IsEquiv.val_le val_isEquiv_comap).symm
```
and `IsValExtension.val_map_lt_iff`, `IsValExtension.val_map_le_one_iff` ... These are simp lemmas for the class `IsValExtension`.

The restating of the definition is to make the name more consistent. When there are `val_lt` and `val_le_one`, it is better to also have a `val_le`.

What do you think about these lemmas?

Reply:
Then I will delete these lemmas and prove `IsValExtension.val_map_lt_iff` directly.